ggml-sycl.cpp

```diff
@@ -3210,6 +3467,31 @@ typedef struct dpct_type_107281 {
 } block_q6_K;
 static_assert(sizeof(block_q6_K) == sizeof(ggml_fp16_t) + 13*QK_K/16, "wrong q6_K block size/padding");
 
+#define QR2_XXS 8
+#define QI2_XXS (QK_K / (4*QR2_XXS))
+typedef struct dpct_type_101531 {
```

Comment:
Do we need ".._101531" or it can be only struct dpct_type_<quant process such as QR2_XS>

Reply:
yes, rename it.